rdf2vec/converters.py

```diff
@@ -2,6 +2,16 @@
 from tqdm import tqdm
 
 def create_kg(triples, label_predicates):
+    """Creates a knowledge graph according to triples and predicates label.
+
+    Args:
+        triples (list): The triples.
+        label_predicates (list): The predicates label.
```

Comment:
"The URI's of the predicates that have to be excluded from the graph to avoid leakage."